docs/evaluate/temporal-cloud/security.mdx

```diff
@@ -54,114 +54,13 @@ The base unit of isolation in a Temporal environment is a [Namespace](/namespace
 Each Temporal Cloud account can have multiple Namespaces.
 A Namespace (regardless of account) cannot interact with other Namespaces.
 Each Namespace is available through a secure gRPC (mTLS) endpoint and an HTTPS (TLS) endpoint.
-You can make these endpoints more secure by routing all communication through AWS PrivateLink.
+
+* [AWS PrivateLink](aws-privatelink.mdx)
```

Comment:
Wondering if we should keep a section:

```
# Private connectivity

Temporal Cloud supports private connectivity to enable you to connects to Temporal Cloud from a segregated cluster. You can find more details per cloud:

- AWS PrivateLink
```

Thoughts?

Reply:
I'm ok with this. Will add.

Reply:
added.